vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -587,13 +595,16 @@ type internal FsiToolWindow() as this =
                  // REVIEW: log error into Trace.
                  // Example errors include no active document.
 
-    let onMLSend (sender:obj) (e:EventArgs) =       
-        sendSelectionToFSI false
+    let onMLSendSelection (sender:obj) (e:EventArgs) =       
```

Comment:
@latkin fair enough :smile: i'll send this pr. 
I previously enabled the unused var warning as error only for fsharp.sln projects, not Deployment.sln